dask_ml/preprocessing/data.py

```diff
@@ -245,7 +245,7 @@ def _check_inputs(self, X, accept_sparse_negative=False):
         # TODO: mix of sparse, dense?
         sample = rng.uniform(size=(5, X.shape[1])).astype(X.dtype)
         super(QuantileTransformer, self)._check_inputs(
-            sample, accept_sparse_negative=accept_sparse_negative
+            sample, accept_sparse_negative=accept_sparse_negative, copy=copy
```

Comment:
Looks like `copy` is only a recognized argument for the development version of Scikit-Learn.

Right now, the CI is throwing errors on every test except `sklearn-dev`. The errors look something like

```
        super(QuantileTransformer, self)._check_inputs(
>           sample, accept_sparse_negative=accept_sparse_negative, copy=copy
        )
E       TypeError: _check_inputs() got an unexpected keyword argument 'copy'
```